lighthouse-core/test/audits/dobetterweb/script-blocking-first-paint-test.js

```diff
@@ -29,6 +32,12 @@ describe('Script Block First Paint audit', () => {
         },
         {
           tag: scriptDetails,
+          transferSize: 100,
+          startTime: 15,
```

Comment:
maybe add a comment indicating that this is after fCP and so expected to not show up? Not really obvious due to different units of time, unfortunately.